### PR TITLE
Add Redis.app v3.0.7-build.1

### PR DESCRIPTION
**Pull request by jpadilla**

### CODE REVIEW

**Comment by jpadilla** [comment]
Not entirely sure why this failed. Look at travis logs I see `/Users/travis/build.sh: line 45: bundle: command not found`.

File: Casks/redis.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+cask 'redis' do
+  version '3.0.7-build.1'
+  sha256 'c0a94b34f913f40cf595dd3b954ab6949ad98c1a83d0b79fa1c95e6ec8366847'
+
+  # github.com is the official download host per the vendor homepage
```
Comment (vitorgalvao) [inline Casks/redis.rb]: [We’re changing the comment](https://github.com/caskroom/homebrew-cask/pull/18051). Should now be `# github.com/jpadilla/redisapp was verified as official when first introduced to the cask`.

File: Casks/redis.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  url "https://github.com/jpadilla/redisapp/releases/download/#{version}/Redis.zip"
+  appcast 'https://jpadilla.github.io/redisapp/appcast.xml',
```
Comment (vitorgalvao) [inline Casks/redis.rb]: We prefer this to follow the github releases page (i.e. `https://github.com/jpadilla/redisapp/releases.atom`).

**Comment by jpadilla** [comment]
@vitorgalvao thanks, made the suggested changes.

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in b2fb78f810b279e0bfdb392544ca95652e5239e8. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**